spec.md

```diff
@@ -325,16 +343,24 @@ on the definition of OPTIONAL.
   event format defines the relationship in
   [section 3.1](./json-format.md#31-handling-of-data).
 
-  When this attribute is omitted, `data` simply follows the event
-  format's encoding rules. For the JSON event format, the `data` value
-  can therefore be a JSON object, array, or value.
+  Some event formats have a default value for `datacontenttype`. Events in such
+  formats MAY omit the `datacontenttype` to imply the default.  For example, in
+  the JSON format `datacontenttype` defaults to "application/json". A
+  JSON-format event with no `datacontenttype` is exactly equivalent to one with
+  `datacontenttype="application/json"`; its `data` can be any JSON value.
+
+  For some binary mode protocol bindings, this field is directly mapped to the
+  respective protocol's content-type metadata property. Normative rules for the
+  binary mode and the content-type metadata mapping can be found in the
+  respective protocol
+
+  Agents that translate messages to a new format or binary-mode protocol message
+  SHOULD propagate the `datacontenttype` from source to target as follows:
 
-  For the binary mode of some of the CloudEvents transport bindings, where the
-  `data` content is immediately mapped into the payload of the transport frame,
-  this field is directly mapped to the respective transport or application
-  protocol's content-type metadata property. Normative rules for the binary mode
-  and the content-type metadata mapping can be found in the respective transport
-  mapping specifications.
+  * Source has a value: copy it to target unmodified.
+  * Source has no value, source format has a default: set the target to the
```

Comment:
I think I can live with this if we can tweak the wording to not use the term "default" per my comment above

Reply:
Pick a word. We want to say "the datacontentype value to be assumed when none is explicitly present".

Reply:
see latest update